### ISSUE TITLE
VectorBender does move the layer in to nowhere

### ISSUE THREAD

**Issue by TobiasSchulthess** [olivierdalang/VectorBender#36]
Hi, I am searching for a way to georeference vectordata using QGIS.
In my search I found this PlugIn, but unfortunately it doesn't work as I hoped to.
When I try to georeference a Polygon it moves it into nomans land.

Am I missing something?

I'm using QGIS 3.18.2-Zürich and LV95 (EPSG:2056) as coordinate system.

![Aufnahme3](https://user-images.githubusercontent.com/99729059/154058327-363e8343-6a31-4836-933b-030640a8c39e.png)
![Aufnahme4](https://user-images.githubusercontent.com/99729059/154058319-837acf15-ab7f-46d3-af15-835b2bcdba47.png)

**Comment by djes** [comment]
Be sure that project and layer CRS are identical.

**Comment by TobiasSchulthess** [comment]
Hi, thanks for replying.
I checked the CRS and they are identical.
Could there be another reason?
Kind regards Tobias
![Aufnahme5](https://user-images.githubusercontent.com/99729059/154062754-6eb26924-6b75-4428-bf5e-09d3a949c565.png)
![Aufnahme6](https://user-images.githubusercontent.com/99729059/154062760-b2500197-9576-46dd-bba9-699a38643930.png)

**Comment by djes** [comment]
Yes, the vector bender layer also has to have the same crs. (You could also
change it with right click)

**Comment by TobiasSchulthess** [comment]
Wow, thank you so much for your help!
You saved my day 💯.
Best greetings Tobias :)

**Comment by djes** [comment]
🙂👍
